CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+3.4.1
+==
+
+## Bug Fixes
+
+ - [Want help to access webview document height and cookies (#5243)](https://github.com/NativeScript/NativeScript/issues/5243)
+ - [Question about plugin using native lib NS 3.4 (#5254)](https://github.com/NativeScript/NativeScript/issues/5254)
+
```

Comment:
it this relevant for the PR?

Reply:
No, I think that was from some merge that accidentally got in there.

Reply:
@DickSmith please rebase to the master branch and drop the other commits. Additionally you can rebase the last 4 commits from the HEAD of your branch to drop the commits manually.

`tns rebase -i HEAD~4`, then for all other commits except yours you type a `d` before the commit sha